steering/elections/2020/VOTERS.md

```diff
@@ -186,3 +190,5 @@ If you feel you have standing in the Istio community but are not listed here, pl
 - zhranklin
 - zqzzq
 - zyfjeff
+
+
```

Comment:
```suggestion

```

MD files should end with only one blank line (I don't know if we have a linter but some care)